test/integration/consul-container/libs/cluster/cluster.go

```diff
@@ -66,7 +66,7 @@ func NewN(t TestingT, conf Config, count int) (*Cluster, error) {
 func New(t TestingT, configs []Config, ports ...int) (*Cluster, error) {
 	id, err := shortid.Generate()
 	if err != nil {
-		return nil, fmt.Errorf("could not generate cluster id: %w", err)
+		return nil, fmt.Errorf("could not cluster id: %w", err)
```

Comment:
Assuming this change wasn't intentional

```suggestion
		return nil, fmt.Errorf("could not generate cluster id: %w", err)
```

Reply:
Actually not sure why this file got touched, just going to revert all of my changes here